Nuno-Ferreira/File_Renaming_APP
# 15/Script/Loop-through-files-and-rename

## Description
Adds the process to rename files and add labels for errors or success.
Closes #15 

## Test Plan
- New test case
- Run `python src/gui.py`
- Click `Rename` -> Observe error
- Choose folder and name sequence -> click `Rename` -> Observe success message
